src/main/frontend/extensions/srs.cljs

```diff
@@ -567,7 +567,6 @@
     (if (seq query-result)
       (let [{:keys [total result]} (query-scheduled repo query-result (tl/local-now))
             review-cards result
-            query-string (if (string/blank? query-string) "All" query-string)
```

Comment:
Source of the bug is shadowing an existing binding. We should stop doing this and use different names. If we want to use the same name, we can add a prefix e.g. `query-string'`